x-pack/plugins/security/common/licensing/license_service.ts

```diff
@@ -16,7 +16,7 @@ export interface SecurityLicense {
   isEnabled(): boolean;
   getType(): LicenseType | undefined;
   getFeatures(): SecurityLicenseFeatures;
-  features$: Observable<SecurityLicenseFeatures>;
+  readonly features$: Observable<SecurityLicenseFeatures>;
```

Comment:
**question** why was the `readonly` keyword required?

Reply:
This is very strange. I could have sworn I changed a type that was marked as `Readonly<SecurityLicense>` to just `SecurityLicense` somewhere, so I made sure the one property in here was readonly, but now looking at the PR diff, I don't see that anywhere! I must be losing my mind, I'll just remove this keyword.